tests/test_urlslug.py

```diff
@@ -188,3 +188,25 @@ def test_removes_apostrophes(self):
 
         expected = '9-4-ohms-law'
         assert expected == slug
+
+    def test_handling_unicode(self):
+        titles = (
+            b'College Physics',
+            u'<span class="os-number">5</span><span class="os-divider"> </span><span class="os-text">'
+            u'Further Applications of Newton’s Laws: Friction, Drag, and Elasticity</span>'.encode('utf-8'),
+            u'<span class="os-text">Introduction: Further Applications of Newton’s Laws</span>'.encode('utf-8')
+        )
+
+        slug = generate_slug(*titles)
+        assert slug == '5-introduction-further-applications-of-newtons-laws'
+
+    def test_handling_bytes(self):
+        titles = (
+            u'College Physics',
```

Comment:
am i seeing this correctly? It says `test_handling_bytes` yet I see the each of the titles begins with unicode.

Reply:
@m1yag1 Yes, it's testing that we can handle `bytes` (the python type) that has unicode in them.

Reply:
:rofl: Ok, you're right, the test names should be the other way round.  Good catch!

Reply:
Updated!